web/jsonapi/jsonapi.go

```diff
@@ -52,6 +52,39 @@ func Data(c *gin.Context, statusCode int, o Object, links *LinksList) {
 	c.Data(statusCode, ContentType, body)
 }
 
+// DataList can be called to send an multiple-value answer with a
+// JSON-API document contains multiple objects.
+func DataList(c *gin.Context, statusCode int, objs []Object) {
```

Comment:
Like `Data`, we should allow to pass `links *LinksList` in parameters. It will be used for pagination, for example.